locale/en/manageiq.po

```diff
@@ -83680,6 +83388,14 @@ msgstr ""
 msgid "object %{name} is not one of %{items}"
 msgstr ""
 
+#: ../app/helpers/shared_helper/ab_show_helper.rb
+msgid "object_attribute"
+msgstr ""
+
+#: ../app/helpers/shared_helper/ab_show_helper.rb
+msgid "object_details"
```

Comment:
Commented in https://github.com/ManageIQ/manageiq-ui-classic/pull/9088 via https://github.com/ManageIQ/manageiq-ui-classic/commit/42f0a989616811f8125e38d3fa08dd8286295893#r139262009